tests/e2e/tests/regression/first-start-form/user-agreements-form.e2e.ts

```diff
@@ -0,0 +1,23 @@
+import { commonUrl } from '../../../helpers/conf';
+import { UserAgreementPage } from '../../../pageObjects';
+import { Common } from '../../../helpers/common';
+
+const userAgreementPage = new UserAgreementPage();
+const common = new Common();
+
+fixture `Agreements Verification`
+    .meta({ type: 'regression' })
+    .page(commonUrl)
+    .requestHooks(common.mock)
+    .beforeEach(async t => {
+        await t.maximizeWindow();
+    });
+test('Verify that user can see message "While adding new visualization plugins, use files only from trusted authors to avoid automatic execution of malicious code." on EULA and Privacy Settings window when he opens the application for the very first time', async t => {
```

Comment:
The name is too long.
Please set a shorter one, if you want to supply all this info, you can add prints within the test

Reply:
The test name is shortened